markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -79,41 +80,50 @@ Stretch your binding too much, and it will spring back too far, causing the main
 
 ![As the curve changes, so should the stretch](knitbinding3.png)
 
-Straight seams exist, and although I've never come across one, maybe uniformly curved seams exist too. But more often than not, the curvature of your seam will vary.
+Straight seams exist, and perhaps uniformly curved seams exist too. But more often than not, the curvature of your seam will vary.
 
 On a typical T-shirt neckline, the curve at the back of your neck is different from the front, and there's probably a somewhat tighter bend behind your shoulder seam.
 
-On a slight curve, the difference between the inner and outer edge is not so big, and a little bit of stretch is all you need. But on a tighter curve, the difference between the inner and outer edge of your binding will be more significant. And you'll need more stretch to accommodate for that.
+On a slight curve, the difference between the inner and outer edge is not so big, and a little bit of stretch is all you need. But on a tighter curve, the difference between the inner and outer edge of your binding will be more significant. And, you'll need more stretch to accommodate for that.
 
-So as you follow your seam, and it curves around necklines, or shoulders, or design features, you will have to adapt the amount of stretch to the situation.
+So as you follow your seam and as it curves around necklines, shoulders, or design features, you will have to adapt the amount of stretch to the situation.
 
 ## You cannot calculate the length of your knit binding
 
-I am not saying it is theoretically impossible. But I am saying that it is practically impossible.
+It may not be theoretically impossible, but it is practically impossible.
 
-The length of your knit binding depends on the amount it needs to be stretched. This depends on the curves you have to follow, and the width of the binding, not to mention the strength of its stretchiness. There's just too many parameters to take into account to calculate your binding length accurately.
+The length of your knit binding depends on the amount it needs to be stretched. This depends on the curves you have to follow and the width of the binding, not to mention the strength of its stretchiness. There are just too many parameters to take into account to calculate your binding length accurately.
 
 ### Do not do this
 
 There are a bunch of tutorials out on the internet that show you how to sew on knit binding.
 Many of those suggest something like _make the neck binding 90% of the length of the seam you are attaching it to_ (the percentage varies).
 
-This idea is that you cut your binding, join its edges, and then sew it in the opening. That is (in my opinion) no good.
+This idea is that you would cut your binding, join its edges to form a closed loop, and then sew it to the opening. However, this is not the best method.
```

Comment:
I have the same issue here as above, before it said the author of this documentation (I assume Joost wrote this) thinks this method is no good. Now the documentation reads as if this is _never_ the right method. I think people should feel free to use the method that works for them and this unequivocal language sends the opposite message. Perhaps we need to change this to 'we' rather than 'I' so it still signals that this is someone's opinion rather than solid fact.

Reply:
What about if we have a section with `Alternative techniques`? It could include this one and others that may not be as reliable as this, which in itself is subjective.